### PR TITLE
python class : add support for type alias

### PR DESCRIPTION
**Pull request by iwanbk**
Parts of #409, originally reported at https://github.com/Jumpscale/go-raml/issues/228#issuecomment-353972595.

The solution is to alias the class, something like
```
Label = str
```

example of the generated class:
(1) alias of builtin type:
- raml : https://github.com/Jumpscale/go-raml/blob/fbd3aeea547b8c173b18abfccc4afe0994b9925c/codegen/fixtures/struct/struct.raml#L140-L142
- class : https://github.com/Jumpscale/go-raml/blob/fbd3aeea547b8c173b18abfccc4afe0994b9925c/codegen/python/fixtures/class/AliasBuiltin.py#L1-L7

(2) alias of non builtin type : 
- raml : https://github.com/Jumpscale/go-raml/blob/fbd3aeea547b8c173b18abfccc4afe0994b9925c/codegen/fixtures/struct/struct.raml#L143-L145
- class : https://github.com/Jumpscale/go-raml/blob/fbd3aeea547b8c173b18abfccc4afe0994b9925c/codegen/python/fixtures/class/Alias.py#L1-L7

### CODE REVIEW

File: codegen/python/fixtures/python_capnp/EnumEnumCityEnumParks.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 from enum import Enum
 
 
-class EnumEnumCityEnumParks:
+class EnumEnumCityEnumParks(EnumWek):
```
Comment (serboctor) [inline codegen/python/fixtures/python_capnp/EnumEnumCityEnumParks.py]: What's EnumWek?

Comment (iwanbk) [reply codegen/python/fixtures/python_capnp/EnumEnumCityEnumParks.py]: My mistake, i was checking that even this file exist in the fixture, it doesn't tested.
I've fixed this and added this file to the test.

**Comment by iwanbk** [comment]
> I think you can also add in the docs somewhere that no capnp will be generated for alias of built-in type and that in the python class there will be no functions to load from/to python/capnp like we have when we generate a gevent client/server.

This PR doesn't include the capnp things, hence the issue description

> Parts of #409, originally reported at #228 (comment).

jsonschema is in progress, it almost finished.

for capnp, this is what i remember from our last chat:
- for non builtin type : we copy all the fields
- for builtin type, no support for it

**Comment by serboctor** [comment]
> This PR doesn't include the capnp things, hence the issue description

Yes, I understand this. But for normal python classes we have a function that loads data from the class to the capnp and vice versa (in case of gevent). However, this wont be the case for python classes that alias a built-in type so I think it needs to be documented somewhere

**Comment by iwanbk** [comment]
> Yes, I understand this. But for normal python classes we have a function that loads data from the class to the capnp and vice versa (in case of gevent). However, this wont be the case for python classes that alias a built-in type so I think it needs to be documented somewhere

This is normal python class, i don't see canpn things there?

https://github.com/Jumpscale/go-raml/blob/fbd3aeea547b8c173b18abfccc4afe0994b9925c/codegen/python/fixtures/class/animal.py

**Comment by serboctor** [comment]
> This is normal python class, i don't see canpn things there?

you're not checking the correct fixtures. Also, the capnp function are added only if the type is gevent since it was part of the gevent task. We can always add it in all cases, but that was never discussed and may be can be part of another task.

https://github.com/Jumpscale/go-raml/blob/fbd3aeea547b8c173b18abfccc4afe0994b9925c/codegen/python/fixtures/python_capnp/Animal.py

**Pull request merged**